lib/node/runner/checker.go

```diff
@@ -369,9 +370,15 @@ func finishBallot(st *storage.LevelDBBackend, b ballot.Ballot, transactionPool *
 		transactions[hash] = tx
 	}
 
-	blk = block.NewBlockFromBallot(b, log)
```

Comment:
Given that you removed this argument from `NewBlockFromBallot` in the previous commit, this means that the previous commit does not compile, does it ?